templates/repo/settings/options.tmpl

```diff
@@ -517,6 +517,12 @@
 								<label>{{ctx.Locale.Tr "repo.pulls.squash_merge_pull_request"}}</label>
 							</div>
 						</div>
+						<div class="field">
+							<div class="ui checkbox">
+								<input name="pulls_allow_fast_forward_only" type="checkbox" {{if or (not $pullRequestEnabled) ($prUnit.PullRequestsConfig.AllowFastForwardOnly)}}checked{{end}}>
+								<label>{{.locale.Tr "repo.pulls.fast_forward_only_merge_pull_request"}}</label>
```

Comment:
At some point, we really need helper texts for these options.
They are tech jargon, and nowhere explained inside Gitea…
However, I don't know if that needs to be done in this PR.

Reply:
Created #28970.

Separately, on this line and a few other places I noticed I was missing the `ctx` before `.locale.Tr`. My original implementation of this predates that, and they didn't all show up as conflicts when I rebased if they were spaced out enough. I think they're all fixed now.